src/components/effects.js

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from 'react'
+import { Intents } from 'cozy-interapp'
+
+export const useRedirectionURL = (client, doctype, options) => {
+  const [fetched, setFetched] = useState(null)
+  useEffect(() => {
+    const fetch = async () => {
+      const intents = new Intents({ client: client })
+      const fetchedRes = await intents.getRedirectionURL(doctype, options)
```

Comment:
If it contains a redirectionURL, can we rename `fetched`, `setFetched` and `fetchedRes` to something like `redirectionURL`, `setRedirectionURL` and `fetchedRedirectionURL`?

Reply:
Done.